src/vs/workbench/parts/terminal/electron-browser/terminalPanel.ts

```diff
@@ -44,22 +42,23 @@ export class TerminalPanel extends Panel {
 		@IThemeService private themeService: IThemeService
 	) {
 		super(TERMINAL_PANEL_ID, telemetryService);
-		this.configurationHelper = new TerminalConfigHelper(platform.platform, this.configurationService, this.themeService);
 		this.toDispose = [];
 	}
 
 	public layout(dimension: Dimension): void {
-		let cols = Math.floor(this.parentDomElement.offsetWidth / TERMINAL_CHAR_WIDTH);
-		let rows = Math.floor(this.parentDomElement.offsetHeight / TERMINAL_CHAR_HEIGHT);
-		if (this.terminal) {
-			this.terminal.resize(cols, rows);
-		}
-		if (this.ptyProcess.connected) {
-			this.ptyProcess.send({
-				event: 'resize',
-				cols: cols,
-				rows: rows
-			});
+		if (this.font.charWidth && this.font.charHeight) {
```

Comment:
That's a matter of taste actually :smile: I prefer one entry point - one exit point in methods.
But to keep to the conventions I'll change it :smile: